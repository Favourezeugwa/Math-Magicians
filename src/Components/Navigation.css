nav {
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: #161b22;
  color: #e1e1e2;
}

.navLogo {
  font-size: 25px;
  font-weight: 700;
  margin: 15px 30px;
}

.navLinks {
  list-style: none;
  display: flex;
  flex-direction: row;
  justify-content: space-around;
  width: 25%;
}

li {
  padding-right: 20px;
  border-right: 2px solid #fff;
  cursor: pointer;
}

a {
  color: #e1e1e2;
}

a:hover {
  color: #ff9523;
}
