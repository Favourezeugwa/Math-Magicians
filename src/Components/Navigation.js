import React from 'react';
import './Navigation.css';
import { Link } from 'react-router-dom';

function Navigation() {
  return (
    <nav>
      <div className="navLogo">
        <h1>Math Magicians</h1>
      </div>
      <ul className="navLinks">
        <li>
          <Link to="/">
            Home
          </Link>
        </li>
        <li>
          <Link to="/calculator">
            Calculator
          </Link>
        </li>
        <li>
          <Link to="/quote">
            Quote
          </Link>
        </li>
      </ul>
    </nav>
  );
}

export default Navigation;
