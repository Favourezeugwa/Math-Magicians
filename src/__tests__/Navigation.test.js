import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navigation from '../Components/Navigation';

it('Test navigation renders correctly', () => {
  const tree = renderer
    .create(<BrowserRouter><Navigation /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
