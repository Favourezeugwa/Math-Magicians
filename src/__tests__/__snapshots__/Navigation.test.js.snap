// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test navigation renders correctly 1`] = `
<nav>
  <div
    className="navLogo"
  >
    <h1>
      Math Magicians
    </h1>
  </div>
  <ul
    className="navLinks"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
